src/routers/alpha-router/alpha-router.ts

```diff
@@ -518,9 +518,14 @@ export class AlphaRouter
         outputBalance.currency,
         TradeType.EXACT_INPUT,
         swapConfig,
-        { ...DEFAULT_CONFIG, ...routingConfig, protocols: [Protocol.V3] } // TODO: Enable V2 once have trade object across v2/v3.
+        { ...DEFAULT_CONFIG, ...routingConfig, protocols: [Protocol.V3] }, // TODO: Enable V2 once have trade object across v2/v3.
```

Comment:
Would be great to add V2 here at some point. It will break a few of your tests but shouldn't be too bad to fix